### ISSUE TITLE
Benchmark table is not accessible

### ISSUE THREAD

**Issue by marco-c** [mbanon/fastspell#15]
It would be nice to have a plain text table instead of a screenshot of a table. The screenshot of a table is not accessible to people who use a screen reader.

**Comment by mbanon** [comment]
Hi @marco-c , thanks for your message. You are right, we'll add an actual table instead of a screenshot as soon as possible.

**Comment by mbanon** [comment]
We replaced the screenshot with a link to the spreadsheet.
Cheers,
-Marta
